### PR TITLE
Tree

### PR DESCRIPTION
**Pull request by Prerna6501**

### CODE REVIEW

File: DiameterOfBinary [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    if(tnode==NULL)
+       return 0;
+       
+    return 1 + max(height(tnode->left), height(tnode->right));  
+}
+
+int max(int a, int b) 
+{ 
+    return (a > b) ? a : b;
+}
```
Comment (Rohit-Vaidya-S) [inline DiameterOfBinary:34]: Single line space after this

File: DiameterOfBinary [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+int diameter(node* tnode)
+{
+
```
Comment (Rohit-Vaidya-S) [inline DiameterOfBinary:37]: Indentation and one line gap

File: HeightOfBinary [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+typedef struct node
+{
+    int value;
+    struct node* left; 
+    struct node* right;
+} node;
+
+node* insert(int value)
+{
+        struct node *newnode=malloc(sizeof(node));
```
Comment (Rohit-Vaidya-S) [inline HeightOfBinary:15]: Too much indented

**Review by Rohit-Vaidya-S — COMMENTED** [review]
Indenation and line gap

File: DiameterOfBinary (hunk 1 above)
Comment (Prerna6501) [reply DiameterOfBinary:34]: ok

**Pull request closed without merging**